paddle/fluid/pybind/inference_api.cc

```diff
@@ -803,6 +803,22 @@ void BindAnalysisConfig(py::module *m) {
            py::arg("mkldnn_int8_enabled_op_types") =
                std::unordered_set<std::string>({}))
       .def("mkldnn_int8_enabled", &AnalysisConfig::mkldnn_int8_enabled)
+      .def("disable_mkldnn_fc_passes",
+           &AnalysisConfig::DisableMkldnnFcPasses,
+           R"DOC(
+           Disable Mkldnn FC
+           Args:
+                None.
+           Returns:
+                None.
+           Examples:
+               .. code-block:: python
+                from paddle.fluid.core import AnalysisConfig
```

Comment:
Is there some other API where it should be put?

Reply:
@sfraczek you can see inference doc for config  https://www.paddlepaddle.org.cn/inference/v2.4/api_reference/python_api_doc/Config/ConfigClass.html


import paddle.inference as paddle_infer

paddle_infer.Config

Reply:
@sfraczek 

There is a configurable API to disable any Pass   `paddle.inference.Config.delete_pass(pass: str)` , see

https://www.paddlepaddle.org.cn/inference/v2.4/api_reference/python_api_doc/Config/OptimConfig.html#ir

Tt is recommended not to add this API if  ‘paddle.inference.Config.delete_pass’ can meet requirements.

Reply:
@qingqing01 
API `disable_mkldnn_fc_passes` was created to disable all fc passes at one go. They were gathered in the method because they strongly depend on each other and it would have very little sense to disable them separately. 
This PR makes fc passes automatically enabled when mkldnn is enabled, so we wanted to give a user the chance to disable them, because previously they were disabled by default. 

There are many methods in current API which work the same way, but instead of disabling multiple passes they enable them (e.g. `enable_mkldnn_bfloat16`, `enable_mkldnn_int8`) . These methods could have been also replaced with many `paddle.inference.Config.insert_pass(pass: str)` calls, but due to the strong connection between passes some dedicated methods were created, and we wanted to do the same with `paddle.inference.Config.disable_mkldnn_fc_passes`.

Reply:
hi @qingqing01 can you please let me know what is the final decision about disable_mkldnn_fc_passes API?